voice-search/readme.md

```diff
@@ -0,0 +1,19 @@
+# Voice Search
+This module includes all essential code to make on-device voice search work.
+
+Voice search is currently available for devices:
+- running API level 33 and above
+- returns true to SpeechRecognizer.isOnDeviceRecognitionAvailable (OEM specific)
+- has en-US locale set
+
+We also have the remote configuration flag voiceSearch which should be able to on the fly change the minimum supported API level
```

Comment:
Should we mention this is currently disabled by default (unless the user had previously used it for retrocompatibility with Pixel 6 and 6 Pro) and that it can be enabled under Settings > Accessibility?